infrastructure/main.tf

```diff
@@ -78,7 +78,7 @@ module "postgresql" {
 
 
   common_tags         = var.common_tags
-  resource_group_name = azurerm_resource_group.rg.name
+  resource_group_name = "rpe-draft-store-flexible-data-${var.env}"
```

Comment:
I think you need to create the resource group before passing this in

Reply:
new Resource group will get created now through the module.